{{ cookiecutter.project_name }}/README.md

```diff
@@ -0,0 +1,177 @@
+[![Bandit](https://img.shields.io/badge/security-bandit-yellow.svg)](https://bandit.readthedocs.io/en/latest/)
+[![codecov](https://codecov.io/gh/{{ cookiecutter.github_username }}/{{ cookiecutter.project_name }}/branch/main/graph/badge.svg)](https://codecov.io/gh/{{ cookiecutter.github_username }}/{{ cookiecutter.project_name }})
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
+[![Documentation Status](https://readthedocs.org/projects/{{ cookiecutter.project_slug }}/badge/?version=latest)](https://{{ cookiecutter.project_slug }}.readthedocs.io/en/latest/?badge=latest)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](./LICENSE)
+[![Python {{ cookiecutter.python_version }}](https://img.shields.io/badge/python-{{ cookiecutter.python_version }}-blue.svg)](https://www.python.org/)
+[![GitHub contributors](https://img.shields.io/github/contributors/{{ cookiecutter.github_username }}/{{ cookiecutter.project_name }})](https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_name }}/graphs/contributors)
+[![Ruff](https://img.shields.io/badge/linter%20&%20formatter-ruff-000000.svg)](https://docs.astral.sh/ruff/)
+[![Safety](https://img.shields.io/badge/security-safety-orange.svg)](https://safetycli.com/product/safety-cli)
+
+# {{ cookiecutter.project_name }}
+
+{{ cookiecutter.short_description }}
+
+## Table of Contents
+
+- [Basic Usage](#basic-usage)
+- [Installation](#installation)
+- [Development](#development)
+- [Contributing](#contributing)
+- [Code of Conduct](#code-of-conduct)
+- [Versioning](#versioning)
+- [License](#license)
+- [Contact](#contact)
+
+## Basic Usage
+
+## Installation
+
+## Development
+
+For ease, certain scripts have been abbreviated in `Makefile`, make sure that
+you have installed the dependencies before running the commands.
+
+> **Note**: `make` commands are only available for Unix-based systems.
+
+To view the commands available, run:
+
+```sh
+make
+```
+
+Here are certain commands that you might find useful:
+
+- Making a virtual environment
```

Comment:
Better: "Creating"

Also, I think it reads more naturally if you change the forms to just "Create [...]", "Install [...]", rather than "Creating", "Installing" etc.